mlir/include/mlir/Target/LLVM/ROCDL/Utils.h

```diff
@@ -71,14 +71,17 @@ class SerializeGPUModuleBase : public LLVM::ModuleToObject {
   virtual std::optional<SmallVector<std::unique_ptr<llvm::Module>>>
   loadBitcodeFiles(llvm::Module &module) override;
 
-  /// Adds `oclc` control variables to the LLVM module.
+  /// Determines required Device Libraries and adds `oclc` control variables to
+  /// the LLVM Module if needed. Also sets
+  /// `amdhsa_code_object_version` module flag
```

Comment:
```suggestion
  /// `amdhsa_code_object_version` module flag.
```

Reply:
Also, could you add  a sentence here saying that `amdhsa_code_object_version` is equal to the ABI value, and that it uses `llvm::Module::Error` to set the flag.